content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -154,7 +154,8 @@ This can also be achieved with Istio `ServiceEntry`, rather than Kubernetes
 `Service`. However, a `ServiceEntry` does not configure the Kubernetes DNS server.
 This means that DNS will need to be configured either manually or
 with automated tooling such as the
-[Istio CoreDNS Plugin](https://github.com/istio-ecosystem/istio-coredns-plugin).
+[Address auto allocation](https://istio.io/latest/docs/ops/configuration/traffic-management/dns-proxy/#address-auto-allocation)
+feature of [Istio DNS Proxying](https://istio.io/latest/docs/ops/configuration/traffic-management/dns-proxy/).
```

Comment:
```suggestion
feature of [Istio DNS Proxying](https://github.com/istio/istio.io/blob/release-1.14/content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md).
```

Reply:
Please see link below

Reply:
Thank you so much for the link. Updated.